test/e2e/framework/util.go

```diff
@@ -260,13 +261,29 @@ func log(level string, format string, args ...interface{}) {
 	fmt.Fprintf(GinkgoWriter, nowStamp()+": "+level+": "+format+"\n", args...)
 }
 
+// Used for important logs, controlled by ginkgo, on by default
 func Logf(format string, args ...interface{}) {
 	log("INFO", format, args...)
 }
 
+// Less important logs, controlled with -v
+var totaldebugs = 0
```

Comment:
I couldn't figure out what the levels really meant, so I used 2 because that way you get e2e logs without lower level client logs. U sure about "4"?